katello/katello-selinux/katello-selinux.spec

```diff
@@ -0,0 +1,148 @@
+# vim: sw=4:ts=4:et
+#
+# Copyright (c) 2013 Red Hat, Inc.
+
+# This program and entire repository is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+%define selinux_variants targeted
+%define selinux_modules katello
+
+%if 0%{?rhel} == 5
```

Comment:
Is EL5 still relevant because of client tools?

Reply:
Correct.